apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts

```diff
@@ -176,21 +176,39 @@ router.post(
         urlPrefix: res.locals.urlPrefix,
         authn_user_id: res.locals.authn_user.user_id,
         user_id: res.locals.user.user_id,
+        mode: 'ungraded',
       });
       res.redirect(res.locals.urlPrefix + '/jobSequence/' + jobSequenceId);
-    } else if (req.body.__action === 'ai_grade_assessment_test') {
+    } else if (req.body.__action === 'ai_grade_assessment_graded') {
       const ai_grading_enabled = await features.enabledFromLocals('ai-grading', res.locals);
       if (!ai_grading_enabled) {
         throw new error.HttpStatusError(403, 'Access denied (feature not available)');
       }
-      const jobSequenceId = await aiGradeTest({
+      const jobSequenceId = await aiGrade({
+        question: res.locals.question,
+        course: res.locals.course,
+        course_instance_id: res.locals.course_instance.id,
+        assessment_question: res.locals.assessment_question,
+        urlPrefix: res.locals.urlPrefix,
+        authn_user_id: res.locals.authn_user.user_id,
+        user_id: res.locals.user.user_id,
+        mode: 'human_graded',
+      });
+      res.redirect(res.locals.urlPrefix + '/jobSequence/' + jobSequenceId);
+    } else if (req.body.__action === 'ai_grade_assessment_all') {
```

Comment:
Nit, AFIACT you now have three branches where the only difference is the `mode` value. Consider collapsing them into a single branch and only changing the value of `mode` as needed.